### PR TITLE
Add support for cursor pagination in AlertListOptions

### PR DESCRIPTION
**Pull request by jporzucek**
Fixes https://github.com/google/go-github/issues/2511

### CODE REVIEW

**Comment by gmlewis** [comment]
Waiting for reply: https://github.com/google/go-github/issues/2511#issuecomment-1290976076

**Review by gmlewis — APPROVED** [review]
Thank you, @jporzucek !
LGTM.

Awaiting LGTM+Approval from any other contributor to this repo before merging.

**Review by valbeat — APPROVED** [review]

**Comment by gmlewis** [comment]
Thank you, @valbeat !
Merging.

**Pull request merged**
